README.md

```diff
@@ -22,121 +22,112 @@ refreshing the code and making it available for easy reuse by others._
-
-    brew install geos
-
-On Windows, consider using MSYS2 to install the library in a Linux-esque manner.
-The [mingw-w64-x86_64-geos](https://packages.msys2.org/package/mingw-w64-x86_64-geos)
-package should be appropriate. If this doesn't work or isn't suitable, cartopy
-or GEOS might have more ideas in their documentation.
-
 ### Running unit tests
 There are a handful of unit tests using pytest, in the [`tests`](tests/)
-directory. These assert some operations and methods used in the stages. They may
+directory. These assert some operations and methods used in the steps. They may
 be run in the regular method:
 
     pytest
 
-### Running stages
+### Running steps
 Execute these commands from the repository root.
 
 See [`docs`](docs/) directory for more details.
 
 MLflow parameters:
 
 * `experiment-name`: "tag" to use for MLflow experiment. Used to share artifacts
-  between stages, i.e. you should run the training stage with a name you used to
-  run the data processing stage.
+  between steps, i.e. you should run the training step with a name you used to
+  run the data processing step.
 * `exp_id`: TODO: one way MLflow distinguishes runs. May need to set to share
-  artifacts between stages...?
+  artifacts between steps...?
 * `run_id`: TODO: one way MLflow distinguishes runs. May need to set to share
-  artifacts between stages...?
+  artifacts between steps...?
 
 For old MLflow versions (TODO: which?), replace the `--env-manager=local` flag
-with `--no-conda`
+with `--no-conda`.
+
+For command-line option explanation, run the appropriate step with `--help` e.g.
+`python src/gz21_ocean_momentum/cli/data.py --help`.
 
 #### Data processing
-The [`cmip26.py`](src/gz21_ocean_momentum/cmip26.py) script runs the data
-processing stage. It generates coarse surface velocities and diagnosed forcings
-from the CM2.6 dataset and saves them to disk. You may configure certain
-parameters such as bounds (lat/lon) and CO2 level.
+The CLI into the data processing step is at
+[`cli/data.py`](src/gz21_ocean_momentum/cli/data.py). It generates coarse
+surface velocities and diagnosed forcings from the CM2.6 dataset and saves them
+to disk. You may configure certain parameters such as bounds (lat/lon) and CO2
+level.
 
 **You must configure GCP credentials to download the CM2.6 dataset used.**
 See [`docs/data.md`](docs/data.md) for more details.
 
-Relevant parameters:
+Example invocation:
 
-* `factor`: the factor definining the low-resolution grid of the generated data
-  with respect to the high-resolution grid.
-* `CO2`: 0 for control, 1 for 1% increase per year dataset.
-* `global`: TODO "make data cyclic along longitude". Set to 0; currently fails when set to 1.
-* `ntimes`: the number of days to process, knowing that the data set is at a
-  time resolution of one per day. If not specified, uses the complete dataset.
-* `lat_min`, `lat_max`, `lon_min`, `lon_max`: the spatial domain to process.
+    python src/gz21_ocean_momentum/cmip26.py \
```

Comment:
Command needs updating, as that file has been deleted

Reply:
When I run this with the cli/data.py command instead I get:

> INFO:__main__:selecting forcing bounding box...
Traceback (most recent call last):
  File "/home/marion/workspace/gz21_ocean_momentum/src/gz21_ocean_momentum/cli/data.py", line 88, in <module>
    forcings = bound_dataset("yu_ocean", "xu_ocean", forcings, bbox)
               ^^^^^^^^^^^^^
NameError: name 'bound_dataset' is not defined

Reply:
Thanks, omitted an import.